README.md

```diff
@@ -156,6 +156,13 @@ Tide's git capabilities are inherited from fish's built-in [fish_git_prompt][].
 | virtual_env_icon    | icon to display in front of virtual_env item                                                                                           | ''         |
 | virtual_env_display | Options are venvName and projectName. venvName is the virtual env directory, most often .venv. projectName is the directory above that | projectName |
 
+### nvm
```

Comment:
I should probably have [Prettier](https://github.com/prettier/prettier) as part of the CI workflow, but could you run Prettier on this file to fix this formatting issues? You can install it as a VSCode extension or as a cli command.

Reply:
This is a pretty cool tool! And I think I will use it more often :smile: I have used it on command line with no options and I think you must be using it simialrily.